scintellometry/meta/observations.py

```diff
@@ -161,7 +170,7 @@ def __init__(self, date, val):
         for k, v in val.iteritems():
             if k == 'ppol' and v.startswith('Polynomial'):
                 self[k] = eval(v)
-            elif k in ('P', 'S', 'channels', 'nodes'):
+            elif k in ('P', 'S', 'channels'):
```

Comment:
Well, actually, thinking more: we just added this for you, right? So, should actually be fine to remove.